src/vs/workbench/api/electron-browser/mainThreadEditor.ts

```diff
@@ -121,7 +121,11 @@ export class MainThreadTextEditor {
 			}));
 
 			let forwardSelection = (event?: ICursorSelectionChangedEvent) => {
-				this._lastSelection = this._codeEditor.getSelections();
+				const newSelections = this._codeEditor.getSelections();
+				if (Selection.selectionsArrEqual(this._lastSelection, newSelections)) {
+					return;
+				}
+				this._lastSelection = newSelections;
```

Comment:
Look valid but also like the wrong place. Shouldn't we make sure that the editor widget isn't firing the event on resize and help all consumer of this event?